packages/nomad/logs-collector.hcl

```diff
@@ -97,9 +97,6 @@ if !exists(.teamID) {
 if !exists(.sandboxID) {
   .sandboxID = "unknown"
 }
-if !exists(.envID) {
```

Comment:
this is duplicated
https://github.com/e2b-dev/infra/pull/393/files#diff-51ef604fac20d65c33eb2fd12ad6ef12873dabb10821d41de16890c2e8c2cf19R88